storybook/stories/components/BlockCard/BlockCard.stories.js

```diff
@@ -24,7 +24,8 @@ export default {
     viewMode: 'docs',
     docs: {
       description: {
-        component: '',
+        component:
+          'Features a drop shadow and a stylized red label, no hover effects. Card used in `BlockCardGroup`.',
```

Comment:
Good catch! I've since fixed the description, and also tweaked the decorators for the three cards so they won't display full-width on their respective docs and canvas pages.